src/utils.rs

```diff
@@ -39,6 +39,15 @@ pub fn print_warning<S: AsRef<str>>(warning: S) {
     }
 }
 
+/// Prints a warning to `stderr` in yellow.
```

Comment:
I would bubble up, yes, if the configuration has an issue, we should tell that and stop the loading.